app/models/salt_handler/minion_orchestration.rb

```diff
@@ -12,16 +12,18 @@ def self.can_handle_event?(event)
     parsed_event_data = JSON.parse event.data
     parsed_event_data["fun"] == "runner.state.orchestrate" &&
       (parsed_event_data["fun_args"].first == "orch.kubernetes" ||
-       parsed_event_data["fun_args"].first["mods"] == "orch.kubernetes")
+       parsed_event_data["fun_args"].first == "orch.update" ||
+       parsed_event_data["fun_args"].first["mods"] == "orch.kubernetes" ||
+       parsed_event_data["fun_args"].first["mods"] == "orch.update")
```

Comment:
You could also go a bit beyond and allow this method to also check that ` parsed_event_data["fun"] == "runner.state.orchestrate"`. So the method call would cover "is this an orchestration result we care about?"